sdks/java/extensions/sql/zetasql/src/test/java/org/apache/beam/sdk/extensions/sql/zetasql/ZetaSQLDialectSpecTest.java

```diff
@@ -4805,6 +4805,93 @@ public void testTVFTumbleAggregation() {
     pipeline.run().waitUntilFinish(Duration.standardMinutes(PIPELINE_EXECUTION_WAITTIME_MINUTES));
   }
 
+  @Test
```

Comment:
nit: Are these really `ZetaSQLDialectSpecTests`? There are no matching compliance tests. Its probably worth moving these tests into their own file.

Reply:
Good point. It can be a good point to move all streaming tests to another place.

Reply:
Have moved all streaming tests to another file.